vault.sh

```diff
@@ -245,9 +245,42 @@ HELP
             *)
                 vault login username=$VAULT_USER
         esac
+    else
+        echo "Already logged into vault"
     fi
 }
 
+vault_login_now() {
```

Comment:
This is interesting and I can see the use for this!
What if we make this simple and unset the VAULT_TOKEN being tested by vault_login? Then we just call vault_login again to do login?

Reply:
Sure, That sounds good.